src/Response.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Utopia\Fetch;
+
+/**
+ * Response class
+ * This class is used to represent the response from the server
+ * @package Utopia\Fetch
+ */
+class Response
+{
+    /**
+     * Response Body
+     *
+     * @var mixed
+     */
+    private $body;
+    /**
+     * Response Headers
+     *
+     * @var array<string, string>
+     */
+    private array $headers;
+    /**
+     * Response Status Code
+     *
+     * @var int
+     */
+    private int $statusCode;
+
+    /**
+     * Response constructor
+     * @param int $statusCode
+     * @param mixed $body
+     * @param array<string, string> $headers
+     * @return void
+     */
+    public function __construct(
+        int $statusCode=200,
+        $body='',
+        array $headers=[],
```

Comment:
I think we should remove all default values here. When building new `Response`, all those 3 should be required (by standard)

Reply:
okay, got it